transport-classes-io_uring/src/main/java/io/netty/channel/uring/IoUringIoHandlerOption.java

```diff
@@ -35,16 +37,19 @@ public int getMaxUnboundedWorker() {
     }
 
     public IoUringIoHandlerOption setRingSize(int ringSize) {
+        ObjectUtil.checkPositive(ringSize, "ringSize");
         this.ringSize = ringSize;
         return this;
     }
 
     public IoUringIoHandlerOption setMaxBoundedWorker(int maxBoundedWorker) {
+        ObjectUtil.checkPositive(maxBoundedWorker, "maxBoundedWorker");
```

Comment:
I think this should be >= 0 as with 0 you can basically disable it

Reply:
Zero doesn't mean disabling this function.
“If the count being passed in is 0, then this command returns the current maximum values and doesn't modify the current setting” from [io_uring_register man](https://manpages.debian.org/unstable/liburing-dev/io_uring_register.2.en.html#IORING_REGISTER_IOWQ_MAX_WORKERS)
The current code does not support the semantics of passing in 0.
So I think it makes sense for us to restrict the input of 0.